app/jobs/send_webhook_job.rb

```diff
@@ -8,7 +8,6 @@ def perform
           if key.to_s =~ /\A\_(\w+)/
             begin
               new_items[$1] = server.message_db.message(value.to_i).webhook_hash
-            rescue Postal::MessageDB::Message::NotFound
```

Comment:
This is clearly serving a purpose. Perhaps it can be refactored, but not removed!